operators/volume_loader/nrrd_loader.cpp

```diff
@@ -234,4 +274,40 @@ bool load_nrrd(const std::string& file_name, Volume& volume) {
   return true;
 }
 
+std::vector<double> parse_vector(std::string str) {
+  std::vector<double> result;
+  trim(str);
+  // ensures the string is surround by parenthesis and then remove them
+  if ((str[0] != '(') || (str[str.length() - 1] != ')')) return result;
+  str = str.substr(1, str.length() - 2);
+
+  std::stringstream ss(str);
+  std::string token;
+  while (std::getline(ss, token, ',')) { result.push_back(std::stod(token)); }
+  return result;
+}
+
+void trim(std::string& str) {
+  str.erase(str.begin(), std::find_if(str.begin(), str.end(), [](unsigned char ch) {
+              return !std::isspace(ch);
+            }));
+  str.erase(
+      std::find_if(str.rbegin(), str.rend(), [](unsigned char ch) { return !std::isspace(ch); })
+          .base(),
+      str.end());
+}
```

Comment:
Aren't spaces already removed in line 67?

Reply:
Line 67 removes all spaces for the parameter names (keys), not the values, right?

Reply:
I missed that. Thanks.